### PR TITLE
Show cluster name in monitor dialog

### PR DESCRIPTION
**Pull request by t-rufang**
You can check the [effect picture](https://microsoft-my.sharepoint.com/:i:/g/personal/rufan_microsoft_com1/ERw9GmoQDOhFhi-f9u2ihkQBAqMdV66dds80nuliZ4f36g?e=sQwTVt)

### CODE REVIEW

**Comment by wezhang** [comment]
ok to test

**Comment by wezhang** [comment]
retest this please

File: Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/SparkServerlessClusterStatesModel.java [hunk 1]
```diff
@@ -29,13 +29,14 @@
...
     private String clusterState = "";
+    @NotNull
+    private String clusterID = "";
 
     @NotNull
     public String getMasterState() {
         return masterState;
     }
 
-    @NotNull
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/SparkServerlessClusterStatesModel.java]: Why to remove this?

Comment (t-rufang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/sparkserverless/serverexplore/SparkServerlessClusterStatesModel.java]: I will revert these :)

**Pull request merged**